qp_woltka/tests/test_woltka.py

```diff
@@ -470,7 +470,8 @@ def test_woltka_syndna_to_array(self):
             f'    bowtie2 -p 8 -x {database} -q '
             '${f} -S $PWD/sams/${sn}.sam --seed 42 --very-sensitive -k 16 '
             '--np 1 --mp "1,1" --rdg "0,1" --rfg "0,1" --score-min '
-            '"L,0,-0.05" --no-head --no-unal --un-gz $PWD/reads/${fn}\n',
+            '"L,0,-0.05" --no-head --no-unal --un '
```

Comment:
Just to confirm, it looks like (besides pathing) the switch also changed from --un-gz to just --un. Is this all right?

Reply:
yup